src/SIL.Harmony/Changes/ChangeContext.cs

```diff
@@ -15,7 +16,7 @@
         Commit = commit;
     }
 
-    public Commit Commit { get; }
+    public Commit Commit { get; } // PROBLEM: Interface is CommitBase. How do I do this?
```

Comment:
you can just change the property type from Commit to CommitBase

Reply:
Done in commit e85caf0